stack/stack.go

```diff
@@ -92,6 +98,10 @@ func ParseYAMLData(fileData []byte, regex string, filter string, envsubst bool)
 		return nil, fmt.Errorf("['%s', '%s'] is the only valid provider for this tool - found: %s", providerName, providerNameLong, services.Provider.Name)
 	}
 
+	if len(services.Version) > 0 && !IsValidSchemaVersion(services.Version) {
+		return nil, fmt.Errorf("%s are the only valid versions for the stack file - found: %s", ValidSchemaVersions, services.Version)
```

Comment:
How does this output look when there are multiple values in the `ValidSchemaVersions` slice?

Reply:
This is what it looks like:
`[1.0 2.0] are the only valid versions for the stack file - found: 1.1`